pkg/kubelet/kubelet.go

```diff
@@ -1356,8 +1369,10 @@ func (kl *Kubelet) initializeModules() error {
 	}
 
 	// Start out of memory watcher.
-	if err := kl.oomWatcher.Start(kl.nodeRef); err != nil {
-		return fmt.Errorf("failed to start OOM watcher %v", err)
+	if kl.oomWatcher != nil {
+		if err := kl.oomWatcher.Start(kl.nodeRef); err != nil {
+			return fmt.Errorf("failed to start OOM watcher %v", err)
```

Comment:
Use `%w` for error wrapping?

Reply:
...and a comma to separate the message from the error, i.e.
```go
return fmt.Errorf("failed to start OOM watcher: %w", err)
```

(I understand it's not your code and you're just moving it around, but while at it it makes sense to fix these nitpicks.

Reply:
updated